habitat_sim/robots/mobile_manipulator.py

```diff
@@ -0,0 +1,322 @@
+from typing import List, Optional, Tuple
+
+import attr
+import magnum as mn
+import numpy as np
+
+import habitat_sim
```

Comment:
```suggestion
from habitat_sim.simulator import Simulator
```

Reply:
Looks like you'll also need to grab other stuff in a similar fashion as the top-level module is being used in other places.